### PR TITLE
feat: invalid lanelet documentation

### PR DESCRIPTION
**Pull request by ahmeddesokyebrahim**
## Description

- This PR is releveant to this [issue](https://github.com/autowarefoundation/autoware/issues/3308) where I implement this [idea](https://github.com/orgs/autowarefoundation/discussions/3275).
- The purpose of this PR is add a new module in node architecture  for invalid lanelet (and noticed other missing modules as well)

- Relevant PRs in different repos: 
   - [autoware_common](https://github.com/autowarefoundation/autoware_common/pull/175) :heavy_check_mark: 
  - [tier4/tier4_autoware_msgs - part I](https://github.com/tier4/tier4_autoware_msgs/pull/76) :heavy_check_mark: 
  - [tier4/tier4_autoware_msgs - part II](https://github.com/tier4/tier4_autoware_msgs/pull/87) :heavy_check_mark: 
  - [autoware_adapi_msgs](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/27) :heavy_check_mark: 
  - [autoware_launch](https://github.com/autowarefoundation/autoware_launch/pull/235) :heavy_check_mark:
  - [autoware_universe](https://github.com/autowarefoundation/autoware.universe/pull/3000) :heavy_check_mark:
  - ~[tier4/tier4_ad_api_adaptor](https://github.com/tier4/tier4_ad_api_adaptor/pull/95)~
  

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by yukkysaito** [comment]
@ahmeddesokyebrahim Thank you for PR :+1: 
I doesn't understand the difference, what did you modify? :pray:

**Comment by ahmeddesokyebrahim** [comment]
Thanks @yukkysaito san for your comment.

> Is it only the addition of invalid_lanelet?

Basically yes, and I added as well some missing modules like `virtual_traffic_light`, `run_out`, and `speed_bump` and removed module that does not exist `virtual_stop_line`.

**Comment by yukkysaito** [comment]
@ahmeddesokyebrahim Thank you for reply :+1: 
LGTM

**Review by yukkysaito — APPROVED** [review]

**Review by yukkysaito — APPROVED** [review]
LGTM

**Comment by mitsudome-r** [comment]
Let's merge this with https://github.com/autowarefoundation/autoware.universe/pull/3000

**Comment by xmfcx** [comment]
overall-node-diagram-autoware-universe drawio

I've just opened the file with Adobe Illustrator and saved it again and it got reduced to 87.3KB from 500KB. Can you put this there instead?

**Comment by mehmetdogru** [comment]
@xmfcx until we decide how are we gonna go with the large sized images can we merge this one as previously @ahmeddesokyebrahim prepared? I mean big size picture with drawio meta deta?

**Comment by xmfcx** [comment]
@ahmeddesokyebrahim, thanks for making the change. But as we discussed in the meeting, having the metadata is very important for the planning team.
So let's keep the larger sized file for now.

And we will try to find a general solution for these files in:
- https://github.com/autowarefoundation/autoware/issues/3466

**Comment by ahmeddesokyebrahim** [comment]
@xmfcx - @mehmetdogru : 
You can have a look, and if you do not have any other comments please squash and merge or just let me know you are OK.

**Review by mehmetdogru — DISMISSED** [review]
LGTM, thanks

**Review by mehmetdogru — CHANGES_REQUESTED** [review]
@ahmeddesokyebrahim could you update[ this file ](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-architecture/planning/index.md) in the repo as well?

[update](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-architecture/planning/index.md#:~:text=map%20(detection%20area)-,Out%20of%20ODD%20area,WIP,-Collision%20Detection%20when)

[add missing modules](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-architecture/planning/index.md#:~:text=behavior_velocity_planner%3A%20calculates%20max%20speed%20based%20on%20the%20traffic%20rules.)

cc: @TakaHoribe

**Review by mehmetdogru — DISMISSED** [review]

File: docs/design/autoware-architecture/planning/index.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ For more details, please refer to the design documents in each package.
   - [_virtual_traffic_light_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_virtual_traffic_light_module/)
   - [_occlusion_spot_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_occlusion_spot_module/)
   - [_run_out_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_run_out_module/)
+  - [_no_drivable_lane_](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_velocity_no_drivable_lane_module/docs/no-drivable-lane-design.md)
```
Comment (mehmetdogru) [inline docs/design/autoware-architecture/planning/index.md:171]: lets change this link with the github.io one. We should wait until https://github.com/autowarefoundation/autoware.universe/pull/3967 is merged and sync process is done

**Review by mehmetdogru — CHANGES_REQUESTED** [review]

File: docs/design/autoware-architecture/planning/index.md (hunk 1 above)
Comment (mehmetdogru) [reply docs/design/autoware-architecture/planning/index.md:171]: @ahmeddesokyebrahim I am afraid you forgot to update this link

Comment (mehmetdogru) [reply docs/design/autoware-architecture/planning/index.md]: Thank you :)

**Pull request merged**